src/transformers/modeling_utils.py

```diff
@@ -1779,9 +1785,10 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
         device_map = kwargs.pop("device_map", None)
         max_memory = kwargs.pop("max_memory", None)
         offload_folder = kwargs.pop("offload_folder", None)
-        offload_state_dict = kwargs.pop("offload_state_dict", None)
+        offload_state_dict = kwargs.pop("offload_state_dict", False)
+        load_in_8bit = kwargs.pop("load_in_8bit", False)
+        int8_threshold = kwargs.pop("int8_threshold", None)
```

Comment:
```suggestion
        int8_threshold = kwargs.pop("int8_threshold", 6.0)
```
We set 6.0 later on so let's use that here.